girder_worker/docker/tasks/__init__.py

```diff
@@ -55,9 +55,17 @@ def _run_container(image, container_args,  **kwargs):
                 % (image, container_args, kwargs))
     try:
         return client.containers.run(image, container_args, **kwargs)
+    except nvidia.NvidiaConnectionError:
+        try:
+            logger.info('Running nvidia container without nvidia support: image: %s' % image)
+            client = docker.from_env(version='auto')
+            return client.containers.run(image, container_args, **kwargs)
+        except DockerException as dex:
```

Comment:
Shouldn't this just be:

```
except DockerException:
    logger.exception('Exception when running without nvidia support.')
```

?

Reply:
When we aren't using an nvidia container or are using one when nvidia support is available, we log a `DockerException` via `logger.error(dex)`.  I wasn't trying to change that behavior.  

In both cases, calling `logger.exception` probably makes more sense; I can change that as part of this PR.

Reply:
Or, perhaps we should make that change in a different PR?

Reply:
Let's go ahead and do it on this one if that's OK.